vendor/assets/stylesheets/dvl/components/sidebar_data.scss

```diff
@@ -44,15 +45,22 @@
 }
 
 .sidebar_data_icon {
-  float: left;
-  width: 1rem;
+  position: absolute;
+  top: 50%;
+  left: 0;
+  margin-top: $rhythm * -3;
```

Comment:
The font size is `2.5rem`, so I believe the `margin-top` needs to be `-1.25rem`, not `-1.5rem` as it is.